### ISSUE TITLE
Distance offset and zoom behaviour

### ISSUE THREAD

**Issue by jaluebbe** [GIScience/Leaflet.Heightgraph#75]
The Heightgraph provides a great overview even for long tracks of several 100km. However, local details of this track may be of interest but are not visible as their length is short compared to the graph. 
An thinkable option to circumvent this could be to create a second Heightgraph window which shows the graph for the selected area only (maybe there are much better solutions). 
To realise this, two  options would be nice to have when initialising Heightgraph: 
1. provide an offset for the distance axis, e.g. to start counting the distance with 112.5km. 
2. provide a callback function when _dragEndHandler is called. It could supply the distance values where the zoomed area starts/ends or even provide a geojson of the selected track segment which could be fed into the second instance of Heightgraph where the distance offset is applied.

**Comment by boldtrn** [comment]
Good idea, thanks for the proposal! To throw some alternatives in the ring: 

When you select a part of the elevation by dragging the mouse, the map zooms to that area. We could also "zoom" the heightgraph. When we zoom out again, the zoom is reset to default. 

An alternative could be to allow showing multiple heightgraphs on one page. This would require to specify a container for the map. Therefore the heightgraph could be shown outside of the map as well, which might be interesting for some use cases as well.

In any case, PRs are welcome :)

**Comment by jaluebbe** [comment]
Zooming the Heightgraph while showing the proper distance axis segment sounds interesting and is exactly what I need.

**Comment by jaluebbe** [comment]
Multiple Heightgraphs on one page could be interesting as well. It could be e.g. a comparison of GPS altitude, barometric and elevation model data. Or viewing surface type and steepness at the same time.

**Comment by nrenner** [comment]
Leaflet.Elevation has a pending PR [#65 Zoom elevation plot](https://github.com/MrMufflon/Leaflet.Elevation/pull/65) that supposedly also zooms the current graph when dragging a zoom range, maybe something to copy from (also MIT License).
